config.py

```diff
@@ -10,12 +12,18 @@
         "topic_slug": "univers-ecospheres",
         "prefix": "www",
         "dsn": os.getenv("DATABASE_URL_PROD", ""),
+        "stats_url": "https://stats.data.gouv.fr/index.php",
+        "stats_site_id": "299",
+        "stats_token": os.getenv("STATS_TOKEN", ""),
     },
     "demo": {
         "universe_name": "ecospheres",
         "topic_slug": "univers-ecospheres",
         "prefix": "demo",
         "dsn": os.getenv("DATABASE_URL", ""),
+        "stats_url": "",
```

Comment:
Better as `None`? (here and below)

Reply:
Yeah, but I would have to handle None values for every conf key, or precisely type which can be None... That's why I settled for this hack.